src/methods/runAllTests.ts

```diff
@@ -0,0 +1,77 @@
+import { TestSuite } from "@isildur-testing/api";
+import { globSync } from "glob";
+if (globSync("**/tsconfig.json", { ignore: ["node_modules/**"] }).length > 0) {
+  require("ts-mocha");
+}
+
+// import mocha after ts-mocha
+import Mocha from "mocha";
+
+export const runAllTests = async (): Promise<TestSuite[]> => {
+  return new Promise((resolve) => {
+    const mocha = new Mocha();
+    const testyFiles = globSync(
+      "**/*.{test,spec}.{js,cjs,mjs,jsx,ts,cts,mts,tsx}",
+      { ignore: ["node_modules/**"] }
+    );
+    const suites: TestSuite[] = [];
+    testyFiles.forEach((file) => {
+      mocha.addFile(file);
+    });
+
+    void mocha.loadFilesAsync().then(() => {
```

Comment:
You might be able to `await` this by placing lines 12 to 20 above the `return new Promise` statement. This way you also won't have to handle a potential rejection as it'll bubble upwards on its own. 

If it doesn't work, you'll need a `.catch(reject)` at the end here